include/git2cpp/blame.h

```diff
@@ -1,23 +1,25 @@
-﻿#pragma once
+#pragma once
 
-#include <memory>
+#include <git2/blame.h>
```

Comment:
Let's leave forward declarations instead of `#include`.

Reply:
You're right.